build/build_edge.sh

```diff
@@ -138,6 +138,7 @@ fi
 if [ -f $SETTINGS_FILE ]; then
     source $SETTINGS_FILE
 fi
+PETALINUX="/proj/petalinux/2020.2/petalinux-v2020.2_1108_1/tool/petalinux-v2020.2-final"
```

Comment:
Can we ask PetaLinux team to publish new path and keep old one as a sym link for some days, so that we can get a window to fix our reference?